### ISSUE TITLE
`io/gaw/test_dms.py` test failure on main-dev

### ISSUE THREAD

**Issue by thorbjoernl** [metno/pyaerocom#1412]
**Describe the bug**
Please provide a clear and concise description of what the bug is.
 - Pyaerocom version: 0.25.0.dev0
 - Computing platform: NA
 - Configuration file (if applicable): NA
 - Error message (if applicable):

Most recent numpy (2.1.3):
```
DeprecationWarning: numpy.core is deprecated and has been renamed to numpy._core. The numpy._core namespace contains private NumPy internals and its use is discouraged, as NumPy internals can change without warning in any release. In practice, most real-world usage of numpy.core is to access functionality in the public NumPy API. If that is the case, use the public NumPy API. If not, you are using NumPy internals. If you would still like to access an internal attribute, use numpy._core.defchararray.
```

```
TypeError: np.char.add() requires both arrays of the same dtype kind, but got dtypes: 'object' and '<U1' (the few cases where this used to work often lead to incorrect results).
```

Happens [here](https://github.com/metno/pyaerocom/blob/7b6616d1ea6b3a47f20625b4d8dacc27b9b97044/pyaerocom/io/gaw/reader.py#L236)

**To Reproduce**
1. Pull most recent main-dev and ensure packages are updated (eg. `pip install -e --upgrade`) 
2. pytest tests/io/gaw/test_dms.py

**Expected behavior**
No error

**Screenshots**

**Additional context**
- Seems unrelated to numpy in itself as an error occurs with a 1.X.Y release as well.
- ~~May be related to changes in iris / scitools.~~
- I suspect the lines where the error happens weren't previously executed as part of the tests, as the np.core->np._core deprecation happened a long time ago.

**Comment by thorbjoernl** [comment]
I installed the exact dependencies from the numpy bump branch and reran the tests on both main-dev and the numpy branch from last week and these tests fail on the former and pass on the latter. I take this to mean that this is not caused by a dependency.

**Comment by heikoklein** [comment]
fixed by #1415
